src/lib/utils.js

```diff
@@ -488,6 +488,24 @@ const validateActorName = (actorName) => {
     }
 };
 
+/**
+ * Checks whether a Makefile target exists in an actor directory, or not
+ *
+ * @param {string} actorDirectory Path to the actor's root directory
+ * @param {string} target The Makefile target for which to chck
+ * @return {bool} True if the specified target exists in the Makefile in the actor directory, otherwise false
+ */
+const checkIfMakefileTargetExists = (actorDirectory, target) => {
+    const makefilePath = path.join(actorDirectory, 'Makefile');
+    if (fs.existsSync(makefilePath)) {
+        const makefileContents = fs.readFileSync(makefilePath, { encoding: 'utf-8' });
+        if (new RegExp(`^${target}:`, 'mg').test(makefileContents)) {
```

Comment:
I wonder why multiline

Reply:
I had to do it so that `^` would match the start of any line in the Makefile, not just the whole string, it didn't work without it.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/multiline